src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {
```

Comment:
The default constructor `CinemaHall()` can be removed as it is not explicitly required and will be provided by Java by default if no other constructors are present.